PopupBridge/build.gradle

```diff
@@ -51,69 +51,12 @@ dependencies {
 
 // region signing and publishing
 
-task javadocs(type: Javadoc) {
```

Comment:
Do we need to bring in Dokka for generating/publishing reference docs?

Reply:
I don't think we are publishing any reference docs for iOS currently. Do we think this is big enough that it's necessary or should dev docs eventually be fine for anything public?

Reply:
Ah I didn't know we didn't have reference docs for Popup Bridge. I'm good omitting them if we didn't have them before.